src/components/card/index.ts

```diff
@@ -0,0 +1,3 @@
+import Card from './content/Card';
+
```

Comment:
import & export others too

Reply:
solved

Reply:
I solved above issues but github dosen't permission to push my code
it says this error 'cant-push-to-remote-branch-cannot-be-resolved-to-branch'